CHANGELOG.md

```diff
@@ -1,5 +1,81 @@
+
+
+
+### Bug Fixes
+
+* clean up state when restarting runtimes to avoid errors ([#8936](https://github.com/confluentinc/ksql/pull/8936)) ([8b26151](https://github.com/confluentinc/ksql/commit/8b26151984118646ae613b9897c223ad7af5a17c))
+* preserve old schema behavior for protobuf wrapped primitives ([#8934](https://github.com/confluentinc/ksql/pull/8934)) ([36485e2](https://github.com/confluentinc/ksql/commit/36485e263d8988e73c8150aaa3c59252fcdedc69))
+* add copious amounts of logging to improve visibility into Gen 2 query's runtimes ([#8811](https://github.com/confluentinc/ksql/pull/8811)) ([8d8a512](https://github.com/confluentinc/ksql/commit/8d8a512f2f6a7d7d667393dbbe43f1da1c1492ad))
+* Add null handling to functions ([#8726](https://github.com/confluentinc/ksql/pull/8726)) ([6117604](https://github.com/confluentinc/ksql/commit/6117604fa424dd051f882458a0c95abcdcd8e170))
+* Adds error handling for nested functions ([#8850](https://github.com/confluentinc/ksql/pull/8850)) ([2a60269](https://github.com/confluentinc/ksql/commit/2a60269cadddc8a11522d103d800ae2a6769d690))
+* Apply the ExtensionSecurityManager to UDAFs ([#8776](https://github.com/confluentinc/ksql/pull/8776)) ([a37688c](https://github.com/confluentinc/ksql/commit/a37688cdd33428c1f5c4e67bbc776835f72b0403))
+* bug preventing decimals in (-1, 1) from being inserted / queried ([#8720](https://github.com/confluentinc/ksql/pull/8720)) ([de8284a](https://github.com/confluentinc/ksql/commit/de8284a127de323567276eb6daa2f963a30a6352))
+* check for the queryId in the Task ID for registering saturation metric ([#8789](https://github.com/confluentinc/ksql/pull/8789)) ([b53c1de](https://github.com/confluentinc/ksql/commit/b53c1dedcbe44671578362d48e14033129e8b305))
+* clean up mistaken topic names ([#8857](https://github.com/confluentinc/ksql/pull/8857)) ([53ddafe](https://github.com/confluentinc/ksql/commit/53ddafe72dd8b2d284e9575cff856680b4280bba))
+* CLI should return non-zero error code on failure ([#8892](https://github.com/confluentinc/ksql/pull/8892)) ([238f4fd](https://github.com/confluentinc/ksql/commit/238f4fdd929541ccdd4aeee45cd331964f73b9b3))
+* coerce property values to correct type ([#8765](https://github.com/confluentinc/ksql/pull/8765)) ([7f7a076](https://github.com/confluentinc/ksql/commit/7f7a07619a6ed28f022980ad078607502f4a1f8d))
+* decrease default number of shared runtimes ([#8889](https://github.com/confluentinc/ksql/pull/8889)) ([87bc4b2](https://github.com/confluentinc/ksql/commit/87bc4b2b9e96b923a19497e0ccb8dc5b4e5adb91))
+* do not include schema id in session config ([#8869](https://github.com/confluentinc/ksql/pull/8869)) ([5ddb852](https://github.com/confluentinc/ksql/commit/5ddb852afc37939baa72d9cf6e55dfde9d1e6400))
+* empty to add future list ([#8902](https://github.com/confluentinc/ksql/pull/8902)) ([bca79ac](https://github.com/confluentinc/ksql/commit/bca79ac5922c25e2ddb8f0fd78263260babf0a1a))

```

Comment:
What's the user-facing impact of this fix? It's not clear from the title. Would be nice to edit or remove. cc @wcarlson5 @jnh5y